test/YieldDistributor.t.sol

```diff
@@ -446,3 +450,295 @@ contract YieldDistributorTest is Test {
         yieldDistributor.castVote(percentages);
     }
 }
+
+contract VotingMultipliersTest is YieldDistributorTest {
+    MockMultiplier public mockMultiplier1;
+    MockMultiplier public mockMultiplier2;
+    NFTMultiplier public nftMultiplier;
+
+    function setUp() public override {
+        super.setUp();
+
+        mockMultiplier1 = new MockMultiplier();
+        mockMultiplier2 = new MockMultiplier();
+    }
+
+    function testAddMultiplier() public {
+        yieldDistributor.addMultiplier(IMultiplier(address(mockMultiplier1)));
+        assertEq(address(yieldDistributor.whitelistedMultipliers(0)), address(mockMultiplier1));
+    }
+
+    function testAddMultiplierRevertAlreadyWhitelisted() public {
+        yieldDistributor.addMultiplier(IMultiplier(address(mockMultiplier1)));
+        vm.expectRevert(IVotingMultipliers.MultiplierAlreadyWhitelisted.selector);
+        yieldDistributor.addMultiplier(IMultiplier(address(mockMultiplier1)));
+    }
+
+    function testRemoveMultiplier() public {
+        yieldDistributor.addMultiplier(IMultiplier(address(mockMultiplier1)));
+        yieldDistributor.removeMultiplier(IMultiplier(address(mockMultiplier1)));
+        vm.expectRevert();
```

Comment:
is there an error message to check for the revert?

Reply:
resolved async